docs/directives/index.rst

```diff
@@ -1,22 +1,32 @@
 Directives
 ==========
 
+Directives for creating and modifying needs:
+
 .. toctree::
+   :caption: Need creation
```

Comment:
I would remove the caption, or set it as sub-title in front of "Directives for creating and modifying needs:"

Currently, it looks like we have two introductions to the toc table:
![image](https://github.com/user-attachments/assets/063ff0e2-f5f9-4997-a1ce-9ec06c10babf)

Reply:
done